site/src/xServices/workspace/workspaceXService.ts

```diff
@@ -271,6 +271,20 @@ export const workspaceMachine = createMachine(
                   },
                 },
               },
+              refreshTemplate: {
```

Comment:
Should this be `refreshTemplateAndStart` instead? It could be a bit misleading since it doesn't _just_ refresh.

Reply:
Ahh, yup I think that'd make sense!